definitions/test-andre/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: TEST
+type: ANDRE
+
+synthesis:
```

Comment:
since this is for testing and you don't need the synthesis capabilities you can remove all this section.

If you are planning to actuall test this I said nothing and you can keep it.
Maybe just in case change the metric name to something even more unique like: andre_testing_1234g